docs/content/gameobject/linking-spatialos-entities.md

```diff
@@ -15,60 +16,61 @@ This document is a guide on how to link a SpatialOS entity with a GameObject usi
+When you are using the MonoBehaviour workflow, you interact with SpatialOS entities in the [Runtime]({{urlRoot}}/content/glossary#spatialos-runtime) by using Readers and Writers. Find out how to use interact with SpatialOS entities in the [Reader and Writers]({{urlRoot}}/content/gameobject/readers-writers).
 
 ## The Creation Feature Module
 
 ### How to use the GameObject Creation Feature Module
-The GameObject Creation Feature Module is a default implementation of how to link SpatialOS entities to GameObjects.
-To enable the Creation Feature Module in your project:
-
-1. Set up your project to use the Creation Feature Module.
-	* In your Unity project’s `Packages` directory, locate the Unity Packages manifest `manifest.json`. Add the Creation Feature Module to the manifest by adding: `"com.improbable.gdk.gameobjectcreation": "file:<path-to-gdk>/workers/unity/Packages/com.improbable.gdk.gameobjectcreation"`
-1. Set up your workers to use the Creation Feature Module when the [WorkerConnector]({{urlRoot}}/content/gameobject/creating-workers-with-workerconnector) initializes them. To do this, when you use the Worker Connector, call `GameObjectCreationHelper.EnableStandardGameObjectCreation(Worker.World)`.
- In your project, create a prefab for any SpatialOS entity you want to represent as a GameObject. Where you store the prefab depends on which worker is going to create the GameObject.
-  * For any worker, use the `Resources/Prefabs/Common/` directory.  
+
+The GameObject Creation Feature Module contains an implementation of how to link GameObjects to SpatialOS entities along with a default implementation of how to create GameObjects.
+To enable the GameObject Creation Feature Module in your project:
+
+1. Set up your project to use the GameObject Creation Feature Module.
+  * In your Unity project’s `Packages` directory, locate the Unity Packages manifest `manifest.json`. Add the GameObject Creation Feature Module to the manifest by adding: `"com.improbable.gdk.gameobjectcreation": "file:<path-to-gdk>/workers/unity/Packages/com.improbable.gdk.gameobjectcreation"`
+1. Set up your workers to use the GameObject Creation Feature Module when the [WorkerConnector]({{urlRoot}}/content/gameobject/creating-workers-with-workerconnector) initializes systems.
+  * To use the default implementation of GameObject creation, in your implementation of `WorkerConnector.HandleWorkerConnectionEstablished`, call `GameObjectCreationHelper.EnableStandardGameObjectCreation(Worker.World)`.
+1. In your project, create a prefab for any SpatialOS entity you want to represent as a GameObject. Where you store the prefab depends on which worker is going to create the GameObject.
```

Comment:
```suggestion
1. In your project, create a prefab for any SpatialOS entity you want to represent as a GameObject. Where you store the prefab depends on which worker type is going to create the GameObject.
```
Is it worker type or worker instance?? It's not clear here.

Reply:
Type is correct